### DISCUSSION TITLE
Orchard Core around the world

### DISCUSSION THREAD

**Discussion by Piedone** [OrchardCMS/OrchardCore#14191] [Show and tell]
This is to show where I've given a talk about Orchard Core recently, on various meetups. If you also gave talks about Orchard, add them here!

- [Tech Valley .NET User Group (TVUG)](https://www.meetup.com/techvalleynetusergroup/events/293339663/)
- [Wisconsin .Net User Group](https://www.meetup.com/wisconsin-net-users-group/events/287246678/) and [online](https://www.meetup.com/wisconsin-net-users-group/events/293095442/)
- [.NET Oxford](https://www.meetup.com/dotnetoxford/events/293103363/)
- [Microsoft Developer User Group Graz](https://www.meetup.com/microsoftdevelopergraz/events/293881924)
- [Wellington .NET User Group](https://www.meetup.com/wellidotnet/events/294114565/)
- [Torino .NET](https://www.meetup.com/torino-net-user-group/events/294522628/)
- [Pittsburgh .NET User Group (PGHDOTNET)](https://www.meetup.com/pghdotnet/events/294731413)
- [Greater Lansing User Group for .NET Developers (GLUGNET)](https://www.meetup.com/glugnet/events/291350637/)
- [Evansville Technology Group](https://www.meetup.com/evansville-technology-group/events/294450995/)
- [.NET Developers Community Singapore](https://www.meetup.com/net-developers-sg/events/295129921/)
- [dotnetMALAGA](https://www.meetup.com/dotnetmalaga/events/296073839/)
- [.NET Cambridge](https://www.meetup.com/dotnetcambridge/events/294557146/)
- [dotnetsheff](https://www.meetup.com/dotnetsheff/events/295028446/)
- [Wisconsin .Net User Group](https://www.meetup.com/wisconsin-net-users-group/events/299499486/) again a year later
- [.NET Developers Community Singapore](https://www.meetup.com/net-developers-sg/events/302612139/) again a year later
- [Hot Topics Developer Group ](https://www.meetup.com/hot-topics-developer-group/events/305579015/)
- [Cloud Champions](https://www.cloudchampion.cz/c/build-modern-net-web-apps-faster-with-orchard-core) ([recording](https://microsoft.cloudchampion.pl/c/build-modern-net-web-apps-faster-with-orchard-core))

I'll add more as they happen.

**Comment by hishamco** [comment]
It's valuable to add OC related talks into our docs

**Reply by Piedone** [reply]
These are the ones that are up on YouTube yet: https://www.youtube.com/watch?v=XMMexfOrWl8&list=PLuskKJW0FhJcF-YxKIRj4IyGVjJEGEkSZ&index=164 and https://www.youtube.com/watch?v=cKhAVWm845o&list=PLuskKJW0FhJf_uqOFbozmquvoKeSgKQG2&index=2

**Comment by hishamco** [comment]
Seems the meetup needs a sign in :(

**Reply by Piedone** [reply]
You can just register if you're interested, it's a simple sign-up :).

**Reply by hishamco** [reply]
Seems I saw the YouTube videos before :)

**Comment by Piedone** [comment]
Done [.NET Developers Community Singapore](https://www.meetup.com/net-developers-sg/events/295129921/).

**Comment by MikeAlhayek** [comment]
Great job @Piedone

**Reply by Piedone** [reply]
Thank you! More to come :).

**Reply by elaurentin** [reply]
Yes, Great Job!

**Reply by Piedone** [reply]
Thank you!

**Comment by Piedone** [comment]
Done [dotnetMALAGA](https://www.meetup.com/dotnetmalaga/events/296073839/).

**Comment by Piedone** [comment]
Done [.NET Cambridge](https://www.meetup.com/dotnetcambridge/events/294557146/).

**Comment by Piedone** [comment]
Done a new one at [dotnetsheff](https://www.meetup.com/dotnetsheff/events/295028446/).

**Comment by Piedone** [comment]
Gave a talk at  [Wisconsin .Net User Group](https://www.meetup.com/wisconsin-net-users-group/events/299499486/).

**Comment by Piedone** [comment]
Gave a talk at [.NET Developers Community Singapore](https://www.meetup.com/net-developers-sg/events/302612139/).

**Comment by Piedone** [comment]
Gave a talk at [Hot Topics Developer Group ](https://www.meetup.com/hot-topics-developer-group/events/305579015/) today.

**Comment by hishamco** [comment]
Are there a video links for such talks?

**Reply by Piedone** [reply]
For some of them, yes. Check them out here: https://www.youtube.com/playlist?list=PLuskKJW0FhJcF-YxKIRj4IyGVjJEGEkSZ This most recent one wasn't recorded.

**Comment by Piedone** [comment]
Gave a talk at [Cloud Champions](https://www.cloudchampion.cz/c/build-modern-net-web-apps-faster-with-orchard-core) on Monday, [here's the recording](https://microsoft.cloudchampion.pl/c/build-modern-net-web-apps-faster-with-orchard-core).

**Reply by hishamco** [reply]
I'm not able to access it. Is the recording on YouTube?

**Reply by Piedone** [reply]
No, it's where I linked.

**Reply by hishamco** [reply]
Seems the countries are limited for the registration :(
